src/settings/mailaddress/MailAddressTableModel.ts

```diff
@@ -182,7 +183,7 @@ export class MailAddressTableModel {
 			(config) => Number(config.nbrOfAliases) > this.userGroupInfo.mailAddressAliases.length,
 		)
 		if (plansWithMoreAliases.length > 0) {
-			throw new UpgradeRequiredError("moreAliasesRequired_msg", plansWithMoreAliases)
+			showPlanUpgradeRequiredDialog(plansWithMoreAliases, "moreAliasesRequired_msg")
```

Comment:
I think we didn't want to show dialogs directly from the model, thus an error. We could return something.